src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/ShippingCategory.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" ?>
+
+<resources xmlns="https://api-platform.com/schema/metadata"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd">
+
+    <resource
+        class="Sylius\Component\Shipping\Model\ShippingCategory"
+        shortName="ShippingCategory"
+    >
+        <attribute name="validation_groups">sylius</attribute>
+        <attribute name="normalization_context">
+            <attribute name="groups">
+                <attribute>shipping_category:read</attribute>
+            </attribute>
+        </attribute>
+        <collectionOperations>
+            <collectionOperation name="get" />
+            <collectionOperation name="post">
+                <attribute name="denormalization_context">
+                    <attribute name="groups">shipping_category:write</attribute>
+                </attribute>
+            </collectionOperation>
+        </collectionOperations>
+        <itemOperations>
+            <itemOperation name="get" />
+            <itemOperation name="put">
+                <attribute name="denormalization_context">
+                    <attribute name="groups">shipping_category:update</attribute>
+                </attribute>
+            </itemOperation>
+            <itemOperation name="delete" />
+        </itemOperations>
+        <property name="id" identifier="true" writable="false" />
+        <property name="code" identifier="false" required="true" />
```

Comment:
```suggestion
        <property name="code" identifier="true" required="true" />
```
According to another entities

Reply:
Not exactly, ShippingCategory is received by id: 
![Screenshot 2020-02-26 at 13 26 38](https://user-images.githubusercontent.com/39232096/75345045-383ebe80-589c-11ea-96ce-d7afdb8fa22a.png)

Reply:
i think all sylius routes are identified by id but following by this conversation https://github.com/Sylius/Sylius/pull/11144#discussion_r384342636 in api resource we identify is by code
you like screens so:
![Screenshot 2020-02-26 at 13 54 05](https://user-images.githubusercontent.com/29897151/75346646-75588000-589f-11ea-99de-76a316b756ff.png)
![Screenshot 2020-02-26 at 13 54 15](https://user-images.githubusercontent.com/29897151/75346650-7689ad00-589f-11ea-914f-abbb8f1efeba.png) ☮️ 
cc @Zales0123

Reply:
As we discussed in the office, we would go with code as an identifier, at least for now 🖖